edx_proctoring/templates/timed_exam/ready_to_submit.html

```diff
@@ -7,7 +7,12 @@
   </h3>
   <p>
     {% blocktrans %}
-      Make sure your answers are ready to be submitted, and then submit your exam. Your exam will then be graded.
+      Make sure that you have selected "Check" or "Final Check" for each problem before you submit your exam.
+    {% endblocktrans %}
+  </p>
+  <p>
+    {% blocktrans %}
+      After you submit your exam, your exam will be graded.
```

Comment:
Nit: each problem is graded at the time that you click "Final Check", isn't it? What does submitting the exam actually do?